web/styles/modal.css

```diff
@@ -112,10 +112,13 @@
 }
 
 .modal__btn {
-    font-size: 0.875rem;
-    padding: 0.5rem 1rem;
+    /* We need the backup values for billing related html files where
+       the following variables are not defined. */
+    font-size: var(--base-font-size-px, 14px);
+    padding: calc(0.5 * var(--rem-intermediate-unit, 16px))
+        var(--rem-intermediate-unit, 16px);
     background-color: hsl(0deg 0% 90%);
-    border-radius: 0.25rem;
+    border-radius: calc(0.25 * var(--rem-intermediate-unit, 16px));
```

Comment:
Having the default value specified here is a bit messy, but not sure what is another option here

Reply:
See my comments on your #30914. You're working way too hard here with `calc()`. Just calculate your em value (show your work as a CSS comment), and use that em value here.

Reply:
Giving this another look. I think it's probably okay to convert `border-radius` to just a pixel value; probably the same with the left/right padding value--as we're not really scaling either of those values elsewhere in the UI (yet).

Other things are probably okay as you've argued in #30914, but I'd really love to prioritize refactoring these over-extended classes soon after 9.0 is out.

Reply:
In that case, we don't need the first two commits and this PR is now independent of #30914 now.